src/utils/resolveLambdaDetailsFromManifest.ts

```diff
@@ -7,7 +7,7 @@ const resolveLambdaDetailFromManifest = async (
   detail: CloudFormation.StackResourceDetail;
   lambdaManifest: LambdaManifestType;
 }> => {
-  const cfn = new CloudFormation();
+  const cfn = new CloudFormation({maxRetries: 10});
```

Comment:
What were the errors you were seeing in your case? Perhaps you have a lot of lambdas, so you were hitting a rate limit?

Reply:
Yes, I have about 30 Lambdas and I got rate limited. So retries solve this.

Reply:
Gotcha!